### ISSUE TITLE
[HOLD for payment 2023-10-05] [$500] IOS - Inconsistent margin above Next button in 2FA Step 2

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#26212]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Go to settings-> Click Security
2. Click Two-factor authentication-> Click Copy
3. Click Next

## Expected Result:
Margin should be consistent

## Actual Result:
Inconsistent margin above Next button on 2FA Step 2 screen

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [x] iOS / native
- [ ] iOS / Safari
- [ ] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.53.2
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/93399543/3d3e7775-2e85-4f10-9bfe-47cde3cd4680

![image (18)](https://github.com/Expensify/App/assets/93399543/518eb1c7-88a5-4aeb-843f-6793b6e33fa3)

**Expensify/Expensify Issue URL:**
**Issue reported by:** @Nathan-Mulugeta
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1684403210538459

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01c6a99b3ba9660c1d</li>
        <li>Upwork Job ID: 1699603208696877056</li>
        <li>Last Price Increase: 2023-09-14</li>
<li>Automatic offers: </li>
<ul>
<li>neonbhai | Contributor | 26761577</li>
<li>Nathan-Mulugeta | Reporter | 26761583</li>
</ul></li>
    </ul>
</details>

**Comment by JmillsExpensify** [comment]
Good one. I agree we should fix the margins and ensure they're consistent. cc @Expensify/design to make sure the spec if clear.

**Comment by shawnborton** [comment]
Hmm I am not entirely sure what's being proposed here, but I think we just need to get rid of the additional padding above the green button, does that sound right? 

I also thought we had a pattern where if the page content is longer than the available viewport, the green button would sit directly below the last bit of page content and not cover it up. For example, take a look at the Display name page, which has the ideal behavior I think:
![amWCdvdhc9](https://github.com/Expensify/App/assets/2319350/d3d4b135-0bd9-4d65-b093-0c6084f26dbd)


Also, it looks like this effects other pages too:
<img width="376" alt="image" src="https://github.com/Expensify/App/assets/2319350/55f7b5f8-cd35-4ab0-869b-a44cb1fc3f80">

**Comment by mananjadhav** [comment]
@shawnborton do we then want to cover the floating button in this issue's scope? or we create another issue for that and only fix the padding here?

**Comment by shawnborton** [comment]
They are not separate issues IMO. If we fix the button behavior to match the contact methods page, the padding should be fixed at the same time.

**Comment by JmillsExpensify** [comment]
Agree with @shawnborton on the solution and next steps.

**Comment by mananjadhav** [comment]
@neonbhai's proposal [here](https://github.com/Expensify/App/issues/26212#issuecomment-1697717190) looks good to me. Apologies for the delay, I  was OOO for past few days.

🎀 👀 🎀 C+ reviewed.

**Comment by bondydaa** [comment]
waiting on PR

**Comment by mananjadhav** [comment]
We added the `FixedFooter` to the page in this [PR](https://github.com/Expensify/App/pull/15330). This is where we could've tested the positioning of the button.

I don't think we need any change in the process, as this as a bug.

**Comment by JmillsExpensify** [comment]
Payment summary:

 - Issue reporter: @Nathan-Mulugeta $50
 - Contributor: @neonbhai $1,000 (incl. urgency bonus)
 - Contributor+: @mananjadhav $1,000 (incl. urgency bonus)

**Comment by JmillsExpensify** [comment]
$1,000 payment approved for @mananjadhav based on above summary.

**Comment by JmillsExpensify** [comment]
Issue reporter and contributor are paid out, no regression tests. Closing this issue.

**Comment by Nathan-Mulugeta** [comment]
Hey @JmillsExpensify this issue was created on Aug 29, prior to bounty adjustment announcement, the reporter bounty should be $250 not $50, thankd.

**Comment by mananjadhav** [comment]
@JmillsExpensify Can you help with the previous questions from the contributors? I am not sure what's the process for the old bounty?

Also I don't see the request being approved on the NewDot.

Lastly @JmillsExpensify I am a bit confused how the amount is $1000. If we consider the old bounty it should be $1500, and if we consider the new bounty it should be $750.

**Comment by flaviadefaria** [comment]
@JmillsExpensify following this [comment](https://github.com/Expensify/App/issues/26212#issuecomment-1751915659) [Nathan-Mulugeta](https://github.com/Nathan-Mulugeta) needs to be paid an extra $200 since the bug was reported under the old bounty. 

@Nathan-Mulugeta I just sent you an offer for the remaining $200 please accept it so that I can pay you.

**Comment by Nathan-Mulugeta** [comment]
Thank you so much @flaviadefaria , I just accepted the offer.

**Comment by flaviadefaria** [comment]
Paid, thanks for your patience!

**Comment by JmillsExpensify** [comment]
@mananjadhav yes it should be $750. I got confused working through that. Given that this was my mistake, I'll honor that amount, which benefits you.

**Comment by JmillsExpensify** [comment]
@neonbhai Your proposal was made on Aug 29th, and your proposal wasn't accepted until later. Given that both those dates come after 25 August, the old rates do not apply.

**Comment by JmillsExpensify** [comment]
In contrast, I agree with @Nathan-Mulugeta that his report came before the new payments were establish – in fact, if you look at the original report it was months ago – he is eligible for the older amounts.
